### DISCUSSION TITLE
FR: SoundCloud as a provider

### DISCUSSION THREAD

**Discussion by realroywalker** [music-assistant/support#517] [Feature requests and ideas]
As the title says, I'm interested to know if there is any possibility to add SoundCloud as a provider for MA - they seem to have an API available: https://developers.soundcloud.com/docs/api/guide#playing

There is a free service tier for SoundCloud which allows access to a bunch of unique artists and mixes, so really hoping it's possible to integrate!

## SOUNDCLOUD IS AVAILABLE https://music-assistant.io/music-providers/soundcloud/

**Comment by WebfinityDesign** [comment]
I like to see this too

**Comment by gieljnssns** [comment]
Soundcloud is available on the MA v2 beta
https://github.com/orgs/music-assistant/discussions/1160

**Reply by realroywalker** [reply]
Great news, thanks for your effort! Will have to get v2 setup and have a play 😆

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
